std/container/array.d

```diff
@@ -265,14 +265,13 @@ if (!is(Unqual!T == bool))
         size_t _capacity;
         T[] _payload;
 
-        // Convenience constructor
```

Comment:
It is a private structure and at least for me these kind of comments - where comment is basically the same as function name - are completely redundant.